README.md

```diff
@@ -127,6 +127,7 @@ in February 2016.
 |[Siegen][siegen-wikipedia]|[City of Siegen][siegen-markets]|
 |[Stralsund][stralsund-wikipedia]|[City of Stralsund][stralsund-markets]|
 |[Suhl][suhl-wikipedia]|[City of Suhl][suhl-markets]|
+|[Tübingen][tübingen-wikipedia]|[City of Tübingen][tübingen-markets]|
```

Comment:
The edit in the README.md is incomplete. If you check the [preview rendering of the file on your branch](https://github.com/alexschnapper/wo-ist-markt.github.io/tree/tue) you see that the table of cities is broken. Please fix this.

![readme](https://cloud.githubusercontent.com/assets/144518/25868982/f341bb0c-34fe-11e7-9ca0-45218c869d50.png)